rcmd/install.go

```diff
@@ -5,6 +5,7 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
+	"github.com/spf13/viper"
```

Comment:
rcmd should not need to ever know about how configuration happens. All functions should be parameterized. Hence, instead should update the install api to take a update bool, that you give in the CLI/otherwise the viper.GetBool result